dev-policy.hcl

```diff
@@ -1,7 +0,0 @@
-path "kv-v2/data/*" {
```

Comment:
Do we want an example of a policy somewhere in the repo? Or does that bog it down too much do you think?

Reply:
Ehhh. I'm thinking about it and this may be more confusing having it in than not.

Reply:
I'm leaning toward no just because it feels like it's introducing an "operator best practice" which this definitely is not, and also our new hello-vault repos will have an example policy in them.